eras/babbage/impl/cardano-ledger-babbage.cabal

```
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-ledger-babbage
-version:            1.5.1.1
+version:            1.6.0
```

Also Babbage changelog need to be updated
```suggestion
version:            1.6.0.0
```